apps/prairielearn/src/admin_queries/courses_without_chunks.ts

```diff
@@ -1,4 +1,9 @@
-import { runLegacySqlAdminQuery } from './util.js';
+import { type AdministratorQuerySpecs, runLegacySqlAdminQuery } from './lib/util.js';
+
+export const specs: AdministratorQuerySpecs = {
+  description: 'Finds all courses that do NOT have chunks yet generated.',
+  params: [],
```

Comment:
Same comment regarding empty params.

Reply:
I think that the empty params should be removed, but let's confirm that with Nathan.